posteriors/sgmcmc/sgnht.py

```diff
@@ -0,0 +1,183 @@
+from typing import Any
+from functools import partial
+import torch
+from torch.func import grad_and_value
+from optree import tree_map
+from optree.integration.torch import tree_ravel
+from dataclasses import dataclass
+
+from posteriors.types import TensorTree, Transform, LogProbFn, TransformState
+from posteriors.tree_utils import flexi_tree_map
+from posteriors.utils import is_scalar, CatchAuxError
+
+
+def build(
+    log_posterior: LogProbFn,
+    lr: float,
+    alpha: float = 0.01,
+    beta: float = 0.0,
+    temperature: float = 1.0,
+    momenta: TensorTree | float | None = None,
+    xi: float = None,
```

Comment:
i would call this x0, more standard notation

Reply:
which greek letter is x0? 👀